app/views/guidances/admin_new.html.erb

```diff
@@ -1,99 +1,64 @@
-								</div>
-								<div class="clearfix"></div>
-							</div>
-						</div>
-						</td>
-					</tr>
-          <tr>
-              <td class="first"><%= _('Published') %></td>
-              <td><div class="div_left_icon_g">
-                      <%= f.check_box :published , as: :check_boxes%>
-                  </div>
-                  <div class="div_right_icon_g">
-                    <%= link_to( image_tag('help_button.png'), '#', "data-toggle": "popover", rel: "popover", 'data-html' => "true", 'data-content' => _("Check this box when you are ready for this guidance to appear on user's plans."))%>
-                  </div>
-              </td>
-          </tr>
-					<tr>
-						<td class="first"><%= _('Guidance group') %></td>
-						<td><div class="div_left_icon_g">
-       						<%= f.collection_select(:guidance_group_id, @guidance_groups,
-								:id, :name, {prompt: false, include_blank: 'None'}, {multiple: false})%>
-							</div>
-							<div class="div_right_icon_g">
-								<%= link_to( image_tag('help_button.png'), '#', "data-toggle": "popover", rel: "popover", 'data-html' => "true", 'data-content' => _('Select which group this guidance relates to.'))%>
-							</div>
-							<div class="clearfix"></div>
-
-						</td>
-					</tr>
-				</table>
-
-			<!-- submit buttons -->
-
-				<div class="move_2_right">
-					<a id="new_guidance_submit" data-toggle="modal" href="#new_guidance_alert_dialog" class="btn btn-primary"><%= _('Save')%></a>
-
-					<%= link_to _('Cancel'), :back, class: 'btn cancel' %>
-				</div>
-
-			<br />
-			 <%= tinymce :content_css => asset_path('application.css') %>
-		<%end%>
-	</div>
+<div class="content">
+  <%= form_for :guidance, url: {action: 'admin_create'}, html: {id: 'edit_guidance_form', class: 'roadmap-form bordered'} do |f| %>
+    <fieldset class="side-by-side">
+      <div class="form-input">
+        <%= f.label :text %>
+        <%= text_area_tag("guidance-text", "", class: "tinymce") %>
+        <div class="inline">
+          <a href="#" data-toggle="popover" rel: "popover" data-html: "true"
+             data-content="<%= _('Enter your guidance here. You can include links where needed.') %>">
+          <span class="fa fa-question-circle"></span></a>
+        </div>
+      </div>
+      <div class="form-input">
+        <%= f.label _('Themes'), for: :theme_ids %>
+        <%= f.collection_select(:theme_ids, @themes,
+                  :id, :title, {prompt: false, include_blank: 'None'}, {multiple: true}) %>
+        <div class="inline">
+          <a href="#" data-toggle="popover" rel: "popover" data-html: "true"
+             data-content="<%= _('Select which theme(s) this guidance relates to.') %>">
+          <span class="fa fa-question-circle"></span></a>
+        </div>
+      </div>
+      <div class="form-input">
+        <%= f.label :published %>

```

Comment:
ah good point. Will add a ticket to go through and swap out labels to the standard html `<label for="guidance_published"><%= _('Published') %></label>`.
See ticket #483